File: sdk/client/java/openapi-test/src/test/java/opensdk/sdk/apis/klaytnDebug/vMStandardTracing/DebugStandardTraceBlockToFileTest.java
package opensdk.sdk.apis.klaytnDebug.vMStandardTracing;

import opensdk.sdk.apis.constant.UrlConstants;
import opensdk.sdk.models.DebugStandardTraceBlockToFileResponse;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;
import org.klaytn.OpenSDK;

import java.io.IOException;

@DisplayName("Debug RPC Test")
public class DebugStandardTraceBlockToFileTest {
    private final OpenSDK sdk = new OpenSDK(UrlConstants.SERVER_URL);

    @Test
    @DisplayName("RPC debug_standardTraceBlockToFile")
    void whenRequestValid_ThenCall200ResponseReturns() throws IOException {
        String blockHash = "0xf1b4df5d4457d4771740887eeb46de3fc26ae4cddf93d69b1b237c2366ff12eb";

        DebugStandardTraceBlockToFileResponse response = sdk.debug.standardTraceBlockToFile(blockHash).send();
        response.getResult();
    }
}

File: web3rpc/sdk/client/javascript/openapi-test/test/debug/vMStandardTracing/DebugStandardTraceBlockToFile.test.js
const OpenSdk = require("opensdk-javascript");
const { expect } = require("@jest/globals");
const { RPC } = require("../../constant");

const sdk = new OpenSdk(new OpenSdk.ApiClient(RPC));

describe('debug_standardTraceBlockToFile API', () => {
    test('should return debug_standardTraceBlockToFile', (done) => {

        let callbackOne = function (error, data, response) {
            expect(error).toBeNull();
            expect(data).toBeDefined();
            expect(data).toBeNull();
            done();
        };

        const blockHash = "0xf1b4df5d4457d4771740887eeb46de3fc26ae4cddf93d69b1b237c2366ff12eb";

        sdk.debug.standardTraceBlockToFile(blockHash, {}, callbackOne);
    });
});

